File: tests/Generator/resource/kotlin/complex/Operation.kt

import com.fasterxml.jackson.annotation.*

open class Operation {
    @JsonProperty("arguments") var arguments: HashMap<String, Argument>? = null
    @JsonProperty("authorization") var authorization: Boolean? = null
    @JsonProperty("description") var description: String? = null
    @JsonProperty("method") var method: String? = null
    @JsonProperty("path") var path: String? = null
    @JsonProperty("return") var return: Response? = null
    @JsonProperty("security") var security: ArrayList<String>? = null
    @JsonProperty("stability") var stability: Int? = null
    @JsonProperty("throws") var throws: ArrayList<Response>? = null
}

File: tests/Generator/resource/ruby/complex/operation.rb
class Operation
  attr_accessor :arguments, :authorization, :description, :method, :path, :_return, :security, :stability, :throws

  def initialize(arguments, authorization, description, method, path, _return, security, stability, throws)
    @arguments = arguments
    @authorization = authorization
    @description = description
    @method = method
    @path = path
    @_return = _return
    @security = security
    @stability = stability
    @throws = throws
  end
end

